### ISSUE TITLE
Bio::Blast exec_genomenet: cannot understand response (RuntimeError)

### ISSUE THREAD

**Issue by ramadis** [bioruby/bioruby#123]
Hey guys,

I believe there was a change in BLAST website, breaking the query method in the Blast module.

Stack trace

```
/Users/rama/bio/vendor/bundle/ruby/2.0.0/gems/bio-1.5.1/lib/bio/appl/blast/genomenet.rb:263:in `exec_genomenet': cannot understand response (RuntimeError)
	from /Users/rama/bio/vendor/bundle/ruby/2.0.0/gems/bio-1.5.1/lib/bio/appl/blast.rb:367:in `query'
	from Ex2.rb:19:in `block (3 levels) in <main>'
	from /Users/rama/bio/vendor/bundle/ruby/2.0.0/gems/bio-1.5.1/lib/bio/io/flatfile.rb:336:in `each_entry'
	from Ex2.rb:16:in `block (2 levels) in <main>'
	from Ex2.rb:15:in `open'
	from Ex2.rb:15:in `block in <main>'
	from Ex2.rb:9:in `each'
	from Ex2.rb:9:in `<main>'
``` 

Ruby version: 2.0.0
BioRuby version: 1.5.1

The code is basically:
```
blast = Bio::Blast.remote('blastp', 'swissprot', '-e 0.0001', 'genomenet')
blast.query(entry)
```

**Comment by ramadis** [comment]
@ngoto were you able to check out this issue?

**Comment by ramadis** [comment]
Apparently the issue is that genomenet started using `https`

**Comment by ngoto** [comment]
Thank you for investigation. It should be fixed in some way.

**Comment by ramadis** [comment]
Just created a PR to fix this issue @ngoto

**Comment by ngoto** [comment]
fixed by 4e3251d2172f58239f103e7edf8f4c351140f378 and 6bfef40ae87099565371abf94cf2cc8bfac76b12
